licensing/src/main/java/io/crate/license/exception/LicenseMetadataParsingException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.license.exception;
+
+import java.util.Locale;
+
+public class LicenseMetadataParsingException extends LicenseException {
+
+    public LicenseMetadataParsingException(String message) {
+        super(String.format(Locale.ENGLISH, "License Metadata parsing error: %s", message));
```

Comment:
For simple concat opperations like this it would be way more efficient to simply use `"...: " + message`

`String.format` is parsing the input string to find the place-holder tokens,
and then uses a `StringBuilder` to inject the correctly formatted arguments.
That's all extra work that is not necessary in this case.